index.html

```diff
@@ -108,7 +108,7 @@ <h3>Quality Photos</h3>
                     <p>You will get professional Photography as part of every package to make sure your best moments are
                         captured beutifully forever.</p>
                 </div>
-                <div class="features__cards">
+                <div class="features__cards fifth__card">
                     <i class="fa fa-users"></i>
                     <h3>World Class Tour Guides</h3>
                     <p>We have the most seasoned and experienced tour guides who know the locations inside out along
```

Comment:
- [ ] As the previous reviewer suggested, kindly record 5 minutes loom video about your project and share the link. This is an important project documentation practice and also remember that it is a project requirement.
- [ ] Kindly include the `partners` section missing on your page to match the original  design

### Your design

![image](https://user-images.githubusercontent.com/98453979/190329473-fccc8394-9006-4c49-95a0-3fa25569a045.png)

### Missing Section

![image](https://user-images.githubusercontent.com/98453979/190329748-c570b97b-ae60-4dde-a79d-457125ecf004.png)

- [ ] In your `about` page (mobile version), when you click on the humburger button, your mobile menu is not showing up. Kindly fix it

![image](https://user-images.githubusercontent.com/98453979/190330245-3ef007c0-0e3a-4d26-acb2-24ce70d883f4.png)

Reply:
Kindly sir, my menu works fine when am clicking the hum burger icon of about menu page